libs/openchallenges/styles/src/lib/_general.scss

```diff
@@ -70,6 +70,53 @@ body {
   gap: 24px;
   padding: 28px 0;
 }
+.card-banner {
+  width: 100%;
+  height: 130px;
+  display: flex;
+  align-items: flex-start;
+}
+.card-icon {
+  margin: 0 3px;
+}
+.card-title {
+  margin: 16px 0;
```

Comment:
Sorry, can you elaborate on what you mean?

Reply:
If I understand correctly, the card title (a `<p>` tag)'s margin should be `16px 0` (defined in `card-title` class), but it's actually using the `0 0 12px` (defined in `mat-typography` class). I believe the styles defined in `mat-typography` has higher priority than the styles in `_general.scss`.

That's why the title seems too close to the avatar, because it's missing the top margin.

Sorry not making this clear.

Reply:
Thanks! 16px spacing between avatar and title looks good to me.